tensorflow/tools/test/run_and_gather_logs.py

```diff
@@ -40,10 +40,11 @@
 # pylint: enable=g-bad-import-order
 # pylint: enable=unused-import
 
-from google.protobuf import text_format
 from tensorflow.core.util import test_log_pb2
 from tensorflow.tools.test import run_and_gather_logs_lib
 
+from google.protobuf import text_format
```

Comment:
That's weird, don't do that.  google comes before tensorflow
lexicographically.

On Wed, Mar 23, 2016 at 1:32 PM, caisq <notifications@github.com> wrote:

> In tensorflow/tools/test/run_and_gather_logs.py
> <https://github.com/tensorflow/tensorflow/pull/1603#discussion_r57231515>:
>
> >  from tensorflow.core.util import test_log_pb2
> >  from tensorflow.tools.test import run_and_gather_logs_lib
> >
> > +from google.protobuf import text_format
>
> Suggested by internal linter.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/1603/files/56914f4bb7dc0aa959f2355c726d435b705555e0#r57231515>
>